src/mapml/handlers/ContextMenu.js

```diff
@@ -697,5 +700,38 @@ export var ContextMenu = L.Handler.extend({
   _onItemMouseOut: function (e) {
     L.DomUtil.removeClass(e.target || e.srcElement, 'over');
     this._hideCoordMenu(e);
+  },
+
+  toggleContextMenuItems: function (options,state) {
+    if (state === "disabled"){
+      if (options === "Toggle Controls"){
+        this._items[8].el.el.disabled = true;
+      }else if (options === "Back"){
+        this._items[0].el.el.disabled = true;
+      }else if (options === "Forward"){
+        this._items[1].el.el.disabled = true;
+      }else if (options === "Reload"){
+        this._items[2].el.el.disabled = true;
+      }
+    }else if(state === "enabled"){
```

Comment:
Be consistent about spaces around javascript keywords.  See previous comment about format.